.idea/.gitignore

```diff
@@ -1,3 +1,4 @@
 # Default ignored files
 /shelf/
 /workspace.xml
+/app/schemas
```

Comment:
Question about this - I saw you added a schema file. Will this being in the gitignore interfere with that and what sort of files are you expecting to see in this directory which you want ignored?

Reply:
This will only be the database version json file, to make future auto migrations possible and easier